llama_stack/apis/benchmarks/benchmarks.py

```diff
@@ -45,22 +51,39 @@ class ListBenchmarksResponse(BaseModel):
 
 @runtime_checkable
 class Benchmarks(Protocol):
-    @webmethod(route="/eval/benchmarks", method="GET")
-    async def list_benchmarks(self) -> ListBenchmarksResponse: ...
-
-    @webmethod(route="/eval/benchmarks/{benchmark_id}", method="GET")
-    async def get_benchmark(
-        self,
-        benchmark_id: str,
-    ) -> Optional[Benchmark]: ...
-
-    @webmethod(route="/eval/benchmarks", method="POST")
+    @webmethod(route="/benchmarks", method="POST")
     async def register_benchmark(
         self,
-        benchmark_id: str,
         dataset_id: str,
-        scoring_functions: List[str],
-        provider_benchmark_id: Optional[str] = None,
-        provider_id: Optional[str] = None,
+        grader_ids: List[str],
+        benchmark_id: Optional[str] = None,
         metadata: Optional[Dict[str, Any]] = None,
-    ) -> None: ...
+    ) -> Benchmark:
+        """
+        Register a new benchmark.
```

Comment:
I think you should explain what a benchmark is and when you should think of this term

Reply:
done